resources/quality/goofoo/tpu/goofoo_far_0.40_tpu_95a_draft.inst.cfg

```diff
@@ -0,0 +1,17 @@
+[general]
+version = 4
+name = Draft Quality
+definition = goofoo_far
+
+[metadata]
+setting_version = 16
+type = quality
+quality_type = draft
+material = goofoo_tpu_95a
+variant = 0.6mm Nozzle
```

Comment:
The name of the file is inconsistent with the variant used. Please rename the file to `resources/quality/goofoo/tpu/goofoo_far_0.60_tpu_95a_draft.inst.cfg`.

Reply:
I have already edited it！